cpp/ql/src/Best Practices/Unused Entities/UnusedStaticFunctions.ql

```diff
@@ -13,6 +13,12 @@
 
 import cpp
 
+predicate possiblyIncompleteFile(File f) {
```

Comment:
Should we mark this as `pragma[noinline]` to prevent the compiler undoing your join-order fix?

Reply:
Done.  I think the optimizer would be in a pretty bad state if it did inline that, mind you!